UI/setup/upgrade_info.html

```diff
@@ -50,14 +63,28 @@ <h2><?lsmb text('Database Management Console') ?></h2>
                             options = ar_accounts
                             value_attr = 'accno'
                             text_attr = 'desc'
+                             required = 1
+                            disabled = AR_DISABLED
 } ?>
 </div>
 <div class="input_row">
 <p>
-  LedgerSMB supports multiple <em>Accounts payable (AP)</em> accounts
-per company. One of those must be the system default. Please select
-your default below in case of multiple. If the list below is empty,
-your database is in an inconsistent state and needs to be fixed first.
+  LedgerSMB supports multiple <em>Accounts payable (AP) </em> accounts
+per company.
+  <?lsmb IF ap_accounts.size > 1 ?>
+    <span>One of those must be the system default.
+   Please select your default below.</span>
+  <?lsmb ELSE; ?>
+    <span>Your AP account has been selected to be the
+    system default. Please review.</span>
+  <?lsmb END ?>
+  <?lsmb AP_DISABLED = "";
+    IF ap_accounts.size <= 1 && !ap_accounts.0.id && lsmb_info.ap > 0; ?>
```

Comment:
how can `lsmb_info.ap` be bigger than `0` when there's no account to hold these transactions?

Reply:
If this would happen in actual practice, I'd suggest "serious database integrity damage (please consult a LedgerSMB professional or start over)"